source/leuven/services/openondemand.rst

```diff
@@ -0,0 +1,261 @@
+Which other files will be present in this folder depends on how you created your new template. If you use the 'New Template' button, it will also contain the job 
+script of the default template, being the CPU job template. If you use the 'Copy Template' button, the contents of the folder will depend on the template you are
+copying from. Use the system template that suit your needs the best when starting out. Once you use this more often, you can also use your own templates to create new
+ones. Any file that is present in that folder, will be copied to your new one as well.
+
+Once you've created the new template folder, you can start customizing it to your needs. You could of course immediately create a new job from it, adapt some
+extra options and launch it (which will be explained in the next chapter), but you can still further customize your template folder first. You can view the files in
+the folder using the Folder Explorer (click 'View Files' on top or 'Open Dir' at the bottom). As explained above, you can edit or remove any file, create new files,
+upload new files... These files will be present in each job you create from this template, which means you fully adapt this to the standard set-up you use for your
+jobs.
+
+Jobs
+~~~~
+
+The functioning of creating jobs is a bit similar to how you create new templates. Whatever method you choose, you will create a new folder for each job, this time
+located at ``$VSC_DATA/ondemand/data/projects/default/``. The job folders will be numbered in the order you have created them. **Do not change this folder name as long as you plan on using it from the job menus, as this will break the linking here.** When removing a job, the folder will be deleted as well. 
+
+To create a job, press the 'New Job' button and choose the option that suits your needs the most. You will get a new item in your job list for each job you've created. 
+Again, you can edit, remove and add files like you want to create a custom job by going to the File Explorer (click 'Edit Files' or 'Open Dir') or by directly clicking 
+the file names. The 'Open Editor' button in the 'Submit Script' overview also allows you to edit the job script directly.
+
+Using the 'Job Options' button, you can add some more specifications to your job:
+
+- Name: this will specify a name in the job composer list. This will not be your job name. The actual job name is the one that will be specified in the job script. If you do not specify a name there, you will see that that job gets the name 'sbatch' in the 'Active Jobs' menu.
+- Cluster: there is no need to change this, as you only can specify 'wice'.
+- Specify job script: if you have multiple job scripts in the directory, you can specify which one to run
+- Account: here you can specify which account to use. **Be aware that this will overwrite the account you might have specified in your job script**
```

Comment:
```suggestion
- Specify job script: if you have multiple job scripts in the directory, you can specify which one to run.
- Account: here you can specify which account to use. **Be aware that this will overwrite the account you might have specified in your job script.**
```